inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/ClusterTagRequest.java

```diff
@@ -55,6 +55,9 @@ public class ClusterTagRequest {
     @Length(min = 1, max = 163840, message = "length must be between 1 and 163840")
     private String extParams;
 
+    @ApiModelProperty(value = "The compression type used for dataproxy and sort side data transmission to reduce the network IO overhead")
+    private String inlongCompressType;
```

Comment:
```suggestion
    private String compressType;
```

Reply:
This name is used to distinguish it from other compress types, such as hive's compress type, and this variable name is used in other components